requirements/code_execution.txt

```diff
@@ -19,6 +19,7 @@ ipython
 # this are requirements for code execution - put all packages that LLM should have access to in here
 numpy
 pandas
+scipy
```

Comment:
let's maybe also add scikit-learn as it might sometimes also be used I imagine?

Reply:
I am not sure if we need it. Qwen with TIR prompt is not using it. Maybe we'll need it later, but for now its kinda useless